mobilecoind/src/conversions.rs

```diff
@@ -315,6 +365,84 @@ mod test {
         assert_eq!(rust, UnspentTxOut::try_from(&proto).unwrap());
     }
 
+    // Test the From<&MemoPayload> for api::DecodedMemo implementation
```

Comment:
Nit: This tests `decode_memo` since you opted to not go with a `From<>` implementation

Reply:
right.. i tried using a from, but i got an orphan rules problem, so i changed it to `decode_memo`. thanks